Makefile

```
@@ -58,6 +58,8 @@ endif
 operator-image: vendor/modules.txt pkg/subctl/operator/common/embeddedyamls/yamls.go
 	operator-sdk build quay.io/submariner/submariner-operator:$(DEV_VERSION) --go-build-args "$(OSDK_BUILD_ARGS)"
 
+images: build operator-image
```

Shouldn’t this be

```suggestion
images: operator-image
```

only?

---

At least in the main repo and Lighthouse, `images` depends on building the binaries.

---

I suspect the Submariner and Lighthouse repos have dependencies on `build` in the `images` target because the individual image targets are insufficiently specified (each image ships binaries, and should depend on those binaries directly). In the operator, the binary shipped in the image is built by the Operator SDK in the `operator-image` recipe.

---

Put another way, meta-targets such as `images` should only depend on what they really want, not targets related to _how_ to build what they want.

---

Okay, so should `clusters` also depend on `build` then? Otherwise it seems we're changing behavior there.